ani-cli

```diff
@@ -243,7 +263,8 @@ play_episode() {
             [ -z "$ANI_CLI_NON_INTERACTIVE" ] && printf "All links:\n%s\nSelected link:\n" "$links"
             printf "%s\n" "$episode"
             ;;
-        mpv*) nohup "$player_function" --force-media-title="${allanime_title}Episode ${ep_no}" "$episode" >/dev/null 2>&1 & ;;
+
+        mpv*) nohup "$player_function" --script="${HOME}/.config/ani-cli/scripts/continue_ep.lua" --force-media-title="${allanime_title}Episode ${ep_no}" "$episode" >/dev/null 2>&1 & ;;
```

Comment:
I'd rather have the script loaded by a more generic way - a script directory maybe. Preferably by not changing the code at all, if there's a way.